cmd/up.go

```diff
@@ -165,8 +166,19 @@ List of components that kubeapps up installs:
 			return err
 		}
 		clientset, err := kubernetes.NewForConfig(config)
+		nss := []string{KubeappsNS, KubelessNS, SystemNS}
+		for {
+			if ok, err := allReady(clientset, nss); err != nil {
+				return err
+			} else if ok {
+				break
+			}
+			fmt.Println()
+			fmt.Println("Checking kubeapps readiness...")
```

Comment:
We would need to give the users a little bit more of feedback. I would should the list of deployments (or something like that) and show their status (ready or not). Also if the screen is cleaned between executions of this loop (like the `watch` command) the information would be clearer.

Reply:
I think it's fine if we keep it simple, they can always use kubectl directly for more information.

We should change this message to `Kubeapps is starting up...`

Reply:
👍 @prydonius